devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/examples/azure-functions-example/java/src/test/java/org/acme/FunctionTest.java

```diff
@@ -0,0 +1,69 @@
+package org.acme;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Can we get rid of the star imports please?

Reply:
Ooops!! Cut/paste from MSFT generated code I missed.